libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/README.md

```diff
@@ -0,0 +1,64 @@
+## ZeroOneX6 Flight Controller
+The ZeroOne X6 is a flight controller manufactured by ZeroOne, which is based on the open-source FMU v6X architecture and Pixhawk Autopilot Bus open source specifications.
+
+## Features:
+- Separate flight control core design.
+- MCU 
+   STM32H753IIK6 32-bit processor running at 480MHz
+   2MB Flash
+   1MB RAM
+- Sensors
+- IMU: 
+   Internal Vibration Isolation for IMUs
+   IMU constant temperature heating(1 W heating power).
+   With Triple Synced IMUs, BalancedGyro technology, low noise and more shock-resistant:
+   IMU1-ICM45686(With vibration isolation) 
+   IMU2-BMI088(With vibration isolation) 
+   IMU3- ICM45686(No vibration isolation)
+- Baro:
+   Two barometers:2 x ICP20100
+  Magnetometer:   Builtin RM3100 magnetometer
+
+## Pinout
+![ZeroOneX6 Pinout](https://github.com/ZeroOne-Aero/ardupilot/blob/zeroOneBootLoader/libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/ZeroOneX6Pinout.jpg "ZeroOneX6")
+
+
+## UART Mapping
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+| Name    | Function |
+| ------- | -------- |
+| SERIAL0 | OTG1     |
+| SERIAL1 | UART7    |
```

Comment:
all uarts should also say if TX/RX or both have DMA assigned

Reply:
We have supplemented all uarts if TX/RX or both have DMA assigned.

Reply:
which UART is Telem3? its protocol has not been set...